crates/provider-wit-bindgen/Cargo.toml

```diff
@@ -12,6 +12,9 @@ edition.workspace = true
 license.workspace = true
 repository.workspace = true
 
+[features]
+debug = []
```

Comment:
What does this do?

Reply:
It feels like `RUST_LOG` would be a better fit for this, at appears that it's only guarding the logging statements printed to stderr

Reply:
yeah the point of this is to guard logging statements -- this feature enables debug to stderr output on the bindgen, so if something goes wrong or the output is unexpected, users have a way to figure out what's wrong.

I thought adding something like `env_logger` or `tracing` was overkill here, but I can rename this feature and add some optional imports if we feel strongly about it.

Reply:
Got it. I don't have a strong opinion here. I'm not familiar with whether there's a convention on how to log in macros, so I'd take your word for it, or see what @thomastaylor312 thinks

Reply:
Resolving this since there is more discussion in the other comment (question here was answered)